README.md

```diff
@@ -1,11 +1,23 @@
-# Receiving Api Posts Lab
+# Receiving API POSTs Lab
 
 ## Objectives
 
-  1. Send POST requests with jQuery
-  2. Parse the requests into new AR objects
+  1. Send form via POST request with jQuery.
+  2. Create resources from an AJAX POST.
+  3. Render and consume JSON responses for created resources.
 
-## Outline
+## Introduction
 
-  * Translate the Product create page into a dynamic jQuery/JSON request
-  * Render the newly created product afterwards
+In this lab, we're going to continue to enhance our product and order
+system, this time updating the product creation page to create and
+display new products without requiring a refresh or redirect.
+
+The solution for the previous lab has been included for you.
+
+## Instructions
+
+1. Update the `new` product page to submit the form via AJAX POST.
+2. Update the product `create` action to render a JSON representation of
+   the product after successfully creating it.
+3. Display the product details of the newly created product on the `new` page without refreshing or redirecting.
+4. Make sure those tests pass!
```

Comment:
Cool -- these instructions are clear but leave enough room so that the student has to think about applying what they just learned in the README.